### ISSUE TITLE
Clicked search results open the page at the remembered position not the result's text position

### ISSUE THREAD

**Issue by redactedscribe** [ludovicchabant/obsidian-remember-file-state#20]
When you click a search result, the page will open at the remembered position of the file. You have to click the search result a second time for the page to scroll up to the position of the search result's text.

If a search result is clicked, the file should open and go straight to the result's position in the file, skipping over the logic to restore the remembered position.

Thanks.

**Comment by ludovicchabant** [comment]
Indeed... I'm not sure how to detect this, though. I have a workaround that sort of works in most cases, but not all cases. I'll take suggestions if anybody has a good idea.
